pkg/api/index.go

```diff
@@ -36,17 +36,17 @@ func (hs *HTTPServer) getProfileNode(c *models.ReqContext) *dtos.NavLink {
 
 	children := []*dtos.NavLink{
 		{
-			Text: "Preferences", Id: "profile-settings", Url: hs.Cfg.AppSubURL + "/profile", Icon: "sliders-v-alt",
+			Text: "Preferences", Id: "profile/settings", Url: hs.Cfg.AppSubURL + "/profile", Icon: "sliders-v-alt",
```

Comment:
You'll also need to update the _keys_ in  the `TRANSLATED_MENU_ITEMS` object in `navBarItem-translations.ts`. You don't need to update the `id` inside the  `defineMessage` calls, just the keys for `profile-settings` and `change-password` in the `TRANSLATED_MENU_ITEMS` object.

Doesn't look like there's a key for notifications though :(

Reply:
fixed